packages/synapse-react-client/src/components/EntityFinder/tree/VirtualizedTree.test.tsx

```diff
@@ -77,12 +79,12 @@ describe('VirtualizedTree tests', () => {
     })
 
     it('General test for entity header', () => {
-      const node = {
+      const node: EntityHeaderNode = {
```

Comment:
Fixed types/lint warnings in this test file and moved it so it will be type checked and linted.

I did not add a new test to capture the fix; the existing code structure is just too hard to test the changed function.